fastlane/Helpers/Symbol.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Symbol.swift
+//  FastlaneRunner
+//
+//  Created by Su Ho on 27/09/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+
+enum Symbol {
+
+    static func uploadToCrashlytics(environment: Constant.Environment) {
+        let buildURL = URL(fileURLWithPath: Constant.buildPath)
+        let dsymURL = buildURL.appendingPathComponent(environment.productName + ".app.dSYM.zip")
```

Comment:
I'm changing the build and dsym path in #358. It's better if you include the change and update to this. 🙏

Reply:
Will rebase on your branch 😁

Reply:
Fixed in https://github.com/nimblehq/ios-templates/pull/357/commits/cf2e794f0decf18972a34a6f91d2c2962b444537